pubtools/pulplib/_impl/schema/unit.yaml

```diff
@@ -95,7 +95,9 @@ definitions:
     properties:
       # Type of the unit
       _content_type_id:
-        const: modulemd
```

Comment:
This doesn't seem to be right at all - you're letting modulemd and modulemd_defaults share the same schema here, but they have a different set of expected fields.

For instance, modulemd_defaults doesn't have a "context" field, yet it's listed here as a required field.  I'd expect this not to work.  If I'm right then it means some autotests are also missing.

Reply:
Yeah, I should've checked this instead of assuming they had the same schema. Fixed in next commit.